src/styles/reset-css.scss

```diff
@@ -0,0 +1,3 @@
+body {
```

Comment:
It's recommended to use a more comprehensive CSS reset or normalize styles for better cross-browser consistency.